src/cache.js

```diff
@@ -181,6 +264,9 @@ async function updateGithubProjectsCache() {
 			extractedData.cards[field.column.toLowerCase().replace(' ', '_')]?.push(field.title);
 		}
 
+		const types = await getRepoType(project.url.split("/")[4], project.title)
+		extractedData.type = types
```

Comment:
```suggestion
		extractedData.types = await getRepoType(project.url.split("/")[4], project.title);
```

Or just set this directly on the object rather than reassigning it here